code/obj/flock/structure/collector.dm

```diff
@@ -1,21 +1,21 @@
-/// # Collector structure
+/////////////////////////////////////////////////////////////////////////////////
+// COLLECTOR
+/////////////////////////////////////////////////////////////////////////////////
 /obj/flock_structure/collector
 	name = "weird lookin' pulsing thing"
 	desc = "Seems to be pulsing."
 	flock_id = "Collector"
 	health = 60
 	resourcecost = 200
 	/// does it draw from the local apc if its strong enough.
-	var/drawfromgrid = 0
+	var/drawfromgrid = FALSE
 	/// is it active?
-	var/active = 0
+	var/active = FALSE
 	/// max range for the thing.
 	var/maxrange = 5
 	/// the tiles its connected to
 	var/list/turf/simulated/floor/feather/connectedto = list()
 
- //needed for passthrough
-	// drones can pass through this, might change this later, as balance point
```

Comment:
useful comment

Reply:
I removed it as there's already a doc comment on /obj/flock_structure for this

Reply:
👍